### DISCUSSION TITLE
Private video shown in the latest view

### DISCUSSION THREAD

**Discussion by ttkrpink** [mediacms-io/mediacms#460] [Q&A]
When one user sets a video as private, its preview picture is shown in the latest. Other users might think that video is accessible. Once they click, they will find out it's private.

Should private videos exclude from the latest view?

**Comment by mgogoulos** [comment]
Hi @ttkrpink that would definitely be an issue, but I don't think it is happening. Can you retry, and if this can be reproduced by you, provide the steps to reproduce? What I do is

a) I upload a video, and set it's status to public. Then as a different user (or as anonymous user) I can see it on the index page, and on the user's media page
b) I set the status to private for this video. Then as a different user (or as anonymous user) I cannot see it on the index page, or on the user's media page

**Reply by ttkrpink** [reply]
I think I figure out the problem. 

My application is to capture video from several cams and video capture cards. From the app GUI, users select whether to public or 'private' the videos. Then the videos are uploaded in the background through API.

As I stated in my other thread, it seems that encoding reset videos' state. So I used a while loop checks the encoding status and reset the video state after encoding is done. After encoding is done, "listable" is also reset. My modified API only changes the state, but not the "listable". So the video gets "state=private", "listable=true".

I hope the state parameter can be added to the media ADD API.
